src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -276,8 +276,12 @@ public ProduceMetrics(final Metrics metrics, final Map<String, String> tags) {
           double availabilitySum = 0.0;
           int partitionNum = _partitionNum.get();
           for (int partition = 0; partition < partitionNum; partition++) {
-            double recordsProduced = metrics.metrics().get(metrics.metricName("records-produced-rate-partition-" + partition, METRIC_GROUP_NAME, tags)).value();
-            double produceError = metrics.metrics().get(metrics.metricName("produce-error-rate-partition-" + partition, METRIC_GROUP_NAME, tags)).value();
+            double recordsProduced = (double) metrics.metrics()
+                .get(metrics.metricName("records-produced-rate-partition-" + partition, METRIC_GROUP_NAME, tags))
+                .metricValue();
+            double produceError = (double) metrics.metrics()
+                .get(metrics.metricName("produce-error-rate-partition-" + partition, METRIC_GROUP_NAME, tags))
+                .metricValue();
```

Comment:
I don't disagree with these changes, but they're not strictly part of rebranding

Reply:
sgtm